docs/db_dev_process/fuzzers/index.rst

```diff
@@ -57,7 +57,6 @@ Programmable Interconnect Points (PIPs)
    :maxdepth: 1
```

Comment:
Why are these others being removed?

Reply:
Those fuzzers do not have `README.md` file.